homeassistant/components/automation/trace.py

```diff
@@ -153,14 +172,22 @@ def _check_size_limit(self):
 @contextmanager
 def trace_automation(hass, unique_id, config, context):
     """Trace action execution of automation with automation_id."""
-    automation_trace = AutomationTrace(unique_id, config, context)
+    automation_trace = AutomationTrace(hass, unique_id, config, context)
     trace_id_set((unique_id, automation_trace.run_id))
 
     if unique_id:
         automation_traces = hass.data[DATA_AUTOMATION_TRACE]
         if unique_id not in automation_traces:
             automation_traces[unique_id] = LimitedSizeDict(size_limit=STORED_TRACES)
         automation_traces[unique_id][automation_trace.run_id] = automation_trace
+        automation_trace.fire_trace_event()
+
+        async def breakpoint_hit():
+            automation_trace.state = "debugged"
```

Comment:
This is a weird verb. Maybe "breakpoint" ?

Reply:
I think this is the term for a process or thread which is being controlled by a debugger.